react-components/src/architecture/base/commands/mocks/MockFilterCommand.ts

```diff
@@ -98,11 +97,11 @@ class FilterItemCommand extends BaseFilterItemCommand {
     return this._color;
   }
 
-  public override setChecked(value: boolean): void {
+  protected override setCheckedCore(value: boolean): boolean {
     if (this._use === value) {
-      return;
+      return false;
     }
     this._use = value;
-    CommandsUpdater.update(this._renderTarget);
+    return true;
```

Comment:
In OOP classic concept, we usually have the setter method to set a value in a parameter or some logic around updating a state of some object property. In this change, you are only returning a boolean based on another value. Is it not better than change the method label? Like: getCheckedCore or something more "getter" style?